packages/server/lib/errors.js

```diff
@@ -1010,6 +1010,16 @@ const getMsgByType = function (type, arg1 = {}, arg2, arg3) {
         ${arg1 ? 'Try installing Node.js 64-bit and reinstalling Cypress to use the 64-bit build.'
         : 'Consider upgrading to a 64-bit OS to continue using Cypress.'}
         `
+    case 'NODE_VERSION_DEPRECATION_SYSTEM':
+      return stripIndent`\
+      Deprecation Warning: ${chalk.yellow(`\`nodeVersion\``)} is currently set to ${chalk.yellow(`\`system\``)} in the ${chalk.yellow(`\`cypress.json\``)} configuration file. As of Cypress v9 the default behavior of ${chalk.yellow(`\`nodeVersion\``)} has changed to always use the version of Node used to start cypress via the cli.
```

Comment:
nit, but i would recommend using `v9.0.0` or `9.0.0` instead of `v9` for consistency. I believe other errors/warnings use the full version segments.  Check around for how other errors/warn use the pattern.

Reply:
At the point the breaking options are validation for the configuration, this value technically could have come from multiple places, [config.json, config.ts, or cypress.config.js](https://github.com/cypress-io/cypress/blob/e33f3e703fa059c5fd171129e0cf184df7e54008/packages/server/lib/configFiles.ts#L1).

The file correct configFile is set [here](https://github.com/cypress-io/cypress/blob/e33f3e703fa059c5fd171129e0cf184df7e54008/packages/server/lib/project-base.ts#L696), which will use the [project_utils's getDefaultConfigFilePath func](https://github.com/cypress-io/cypress/blob/e33f3e703fa059c5fd171129e0cf184df7e54008/packages/server/lib/project_utils.ts) to determine which cypress configuration file is found or it falls back to cypress.json.

Can we pass the configFie used into the error message as an additional argument?

Reply:
@emilyrohrbough, good idea. I can make that flex.